src/navigations/root-stack.js

```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 /* eslint-disable react/display-name */
 import NetInfo from '@react-native-community/netinfo';
 import PropTypes from 'prop-types';
-import {SafeAreaView, View} from 'react-native';
+import {SafeAreaView, StatusBar, View} from 'react-native';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `StatusBar` has been added. This is a good change as it allows the application to control the appearance of the status bar.

```diff
+ import {SafeAreaView, StatusBar, View} from 'react-native';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->